scriptworker/cot/generate.py

```diff
@@ -111,9 +118,14 @@ def generate_cot(context, path=None):
     )
     validate_json_schema(body, schema, name="chain of trust")
     body = format_json(body)
-    path = path or os.path.join(context.config['artifact_dir'], "public", "chainOfTrust.json.asc")
+    asc_path = path or os.path.join(context.config['artifact_dir'], "public", "chainOfTrust.json.asc")
+    unsigned_path = os.path.join(os.path.dirname(asc_path), 'chain-of-trust.json')
+    write_to_file(unsigned_path, body)
     if context.config['sign_chain_of_trust']:
+        ed25519_signature_path = "{}.sig".format(unsigned_path)
```

Comment:
Easy enough.

Reply:
This appears to have changed the filename to `chain-of-trust.sig` instead of `chain-of-trust.json.sig`, which isn't what we want. I'm probably going to revert this change.